sdk/metric/aggregation/aggregation.go

```diff
@@ -164,3 +164,65 @@ func (h ExplicitBucketHistogram) Copy() Aggregation {
 		NoMinMax:   h.NoMinMax,
 	}
 }
+
+// ExponentialHistogram is an aggregation that summarizes a set of
+// measurements as an histogram with buckets widths that grow exponentially.
+type ExponentialHistogram struct {
+	// MaxSize is the maximum number of buckets to use for the histogram.
+	MaxSize int
+	// MaxScale is the maximum resolution scale to use for the histogram.
+	MaxScale int
+	// ZeroThreshold sets the minimum value blow which will be recorded as zero.
+	ZeroThreshold float64
+
+	// NoMinMax indicates whether to not record the min and max of the
+	// distribution. By default, these extrema are recorded.
+	//
+	// Recording these extrema for cumulative data is expected to have little
+	// value, they will represent the entire life of the instrument instead of
+	// just the current collection cycle. It is recommended to set this to true
+	// for that type of data to avoid computing the low-value extrema.
+	NoMinMax bool
+}
+
+// DefaultExponentialHistogram returns the default Exponential Histogram aggregation used by the SDK.
+func DefaultExponentialHistogram() ExponentialHistogram {
+	return ExponentialHistogram{
+		MaxSize:       160,
+		MaxScale:      20,
+		ZeroThreshold: 0.0,
+	}
+}
```

Comment:
Could you please explain why we need this? 

> the default Exponential Histogram aggregation used by the SDK.

I do not see it used anywhere outside tests.

Reply:
I added this because if someone used an empty `ExponentialHistogram{}`, they would get a scale of zero, which is valid, but not the best resolution.

These are the values recommended by the spec, and the values that would be used if we implemented the defaulting to exponential, like: [otlp DEFAULT_HISTOGRAM_AGGREGATION](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-configuration).

Reply:
Make sense maybe we can simply change the comment a little e.g.

```suggestion
// DefaultExponentialHistogram returns the default  Exponential Histogram aggregation.
func DefaultExponentialHistogram() ExponentialHistogram {
	return ExponentialHistogram{
		MaxSize:       160,
		MaxScale:      20,
		ZeroThreshold: 0.0,
	}
}
```
The "used by the SDK." is a little confusing for me.